apps/condo/domains/acquiring/schema/RegisterMultiPaymentForOneReceiptService.test.js

```diff
@@ -395,17 +390,13 @@ describe('RegisterMultiPaymentForOneReceiptService', () => {
     describe('ServerSchema get all should provide enough fields', () => {
         test('AcquiringIntegration', async () => {
             const admin = await makeLoggedInAdminClient()
-            const [firstBilling] = await createTestBillingIntegration(admin)
-            const [secondBilling] = await createTestBillingIntegration(admin)
-            const [acquiring] = await createTestAcquiringIntegration(admin, [firstBilling, secondBilling])
+            const [acquiring] = await createTestAcquiringIntegration(admin)
             const [serverObtainedAcquiring] = await AcquiringIntegration.getAll(admin, {
                 id: acquiring.id,
             })
             expect(serverObtainedAcquiring).toBeDefined()
             expect(serverObtainedAcquiring).toHaveProperty('id')
             expect(serverObtainedAcquiring).toHaveProperty('canGroupReceipts')
-            expect(serverObtainedAcquiring).toHaveProperty('supportedBillingIntegrations')
```

Comment:
Why groups are not expected here?

Reply:
`RegisterMultiPayment` mutation depends on `ServerSchema`, so it should be here!

Reply:
Done